src/Components/Login.js

```diff
@@ -48,7 +48,8 @@ class Login extends Component {
             </div>
 
             <button className="btn" type="submit" >Login</button>
-            {this.props.loginSuccess && <Redirect to='/'/>}
+            <p>{this.props.error}</p>
+            {this.props.loginSuccess && <Redirect to='/Feed'/>}
```

Comment:
Put it to the top of render.
You don't need to render the form if login succesed, you just need to redirect